docs/geps/gep-01.rst

```diff
@@ -33,11 +33,16 @@ a nutshell and without explanations, these conventions are:
 * Names follow standard Python conventions (``lowercase_with_underscores``).
   Abbreviations of words that form a part of these names are always followed by an
   underscore, unless it is the last word.
-* Names should be long enough to be readable, but
+* Names should be long enough to be readable. However, we impose limits in order to make
+  GETTSIM usable in languages, which place limits on characters (Stata, in particular).
 
-  - for column names in the user-facing API, there is a hard limit of 17 characters;
-  - for other column names, there is a soft limit of 17 and a hard limit of 25
-    characters.
+  - Column names that are typically user-facing have a hard limit of 20 characters.
+    These columns are documented in `DEFAULT_TARGETS` in `gettsim/config.py`.
```

Comment:
```suggestion
    These columns include input variables and default targets (documented in `STANDARD_DATA_TYPES` and `DEFAULT_TARGETS` in `gettsim/config.py`).
```

Reply:
We should probably rename `STANDARD_DATA_TYPES`

Reply:
Indeed. Suggestions?